Documentation/release-notes/4054.md

```diff
@@ -0,0 +1,24 @@
+## Support for Binding Kotlin Libraries
+
+This release adds better support for creating bindings for Kotlin libraries, including:
+
+* Hiding Kotlin `internal` types and members
+* Using Kotlin provided metadata to properly name method parameters
+* Hiding Kotlin generated methods like `foo-impl` that aren't intended for public use
+* Restoring Kotlin-mangled method names to their original name (ie: `foo-V5j3Lk8` -> `foo`)
+* Adding support for Kotlin's native unsigned types (UInt, ULong, UShort, UByte)
+
+Although it was technically possible to bind a Kotlin library previously, these changes
+greatly reduce the amount of custom `metadata.xml` needed to produce expected bindings.
```

Comment:
```suggestion
greatly reduce the number of custom rules needed in the *Transforms\\Metadata.xml* file to produce expected bindings.
```

(Somewhat off-topic, just in case you might wonder, I've been using backslashes in paths by default in the release notes because even on macOS, items in the `.csproj` file will often use backslashes. I'd be fine with changing that default, especially if something else will work better as more authors start to submit release notes.)